Atlas/backend/README.md

```diff
@@ -72,3 +72,83 @@ the project has the same database schemas.
 ```bash
 ./manage.py runserver 0.0.0.0:8000
 ```
+
+## LINUX
+
+### Refresh your local package index before starting
+```bash
+sudo apt-get update
+```
+
+### Install Python3
+
+[https://www.python.org/downloads/](https://www.python.org/downloads/)
+
+or alternatively
+
+```bash
+sudo add-apt-repository ppa:deadsnakes/ppa
+sudo apt-get update
+sudo apt-get install python3.x
+```
+### Install pip by typing
+```bash
+sudo apt-get install python3-pip
+```
+
+### Once pip is installed, you can use it to install the virtualenv package.
+```bash
+sudo pip3 install virtualenv
+```
+
+### Go to 'backend' directory
+```bash
+cd Atlas/backend
+```
+
+### Create the virtual environment with python3
+```bash
+virtualenv --python=/usr/bin/python3.x atlasenv
```

Comment:
We are using the latest version `3.6`. I think we better either make it `3.6` or annotate it so that it's clear that user should replace `3.x` with his/her python version